plugins/techdocs-backend/src/default-branch.ts

```diff
@@ -42,9 +42,11 @@ interface IGitlabBranch {
   };
 }
 
-function getGithubApiUrl(url: string): URL {
+function getGithubApiUrl(config: Config, url: string): URL {
   const { protocol, owner, name } = parseGitUrl(url);
-  const apiBaseUrl = 'api.github.com';
+  const apiBaseUrl =
+    config.getOptionalString('catalog.processors.github.apiBaseUrl') ||
```

Comment:
This is the old deprecated location. These are now under `integrations` (see the app-config in master for examples)

Reply:
Updated :)